src/olympia/abuse/models.py

```diff
@@ -14,6 +14,41 @@
 
 @python_2_unicode_compatible
 class AbuseReport(ModelBase):
+    # Note: those choices don't need to be translated for now, the
+    # human-readable values are only exposed in the admin. The values will be
+    # updated once they are finalized in the PRD.
+    ADDON_SIGNATURE_CHOICES = (
+        (None, 'None'),
+    )
+    REASON_CHOICES = (
+        (None, 'None'),
+        (1, 'Malware'),
+        (2, 'Spam / Advertising'),
+        (3, 'Search takeover'),
+        (4, 'New tab takeover'),
+        (5, 'Breaks websites'),
+        (6, 'Offensive'),
+        (7, 'Doesn\'t match description'),
+        (8, 'Doesn\'t work'),
+    )
+    REASON_CHOICES_API = {
+        None: None,
+        1: 'malware',
+        2: 'spam_or_advertising',
+        3: 'search_takeover',
+        4: 'new_tab_takeover',
+        5: 'breaks_websites',
+        6: 'offensive',
+        7: 'does_not_match_description',
+        8: 'does_not_work',
+    }
```

Comment:
I wonder if we should be using constants for values `None` to `8` but as this may change back and forth with the PRD I think it's fine for now. Maybe keep that in mind?

Reply:
Yeah... none of the choices are final at this point so I didn't want to go too far with the constants yet... Also, we really should be using something like https://github.com/twidi/django-extended-choices

Reply:
> django-extended-choices

This sounds great, let's do this at some point. Might cleanup our code a lot